app/script/view_model/RightViewModel.coffee

```diff
@@ -57,6 +58,11 @@ class z.ViewModel.RightViewModel
         @conversation_input.removed_from_view()
         @conversation_titlebar.removed_from_view()
 
+    @is_multitasking.subscribe (is_multitasking) =>
+      if is_multitasking
+        amplify.publish z.event.WebApp.ANALYTICS.EVENT, z.tracking.EventName.CALLING.MINIMIZED_FROM_FULLSCREEN,
+        conversation_type: if @call_center.joined_call().is_group() then 'group' else 'one_to_one'
```

Comment:
We should make `group` and `one_to_one` a constant as there are multiple `then 'one_to_one' else 'group'` statements in our code.